sdk/python/v1beta1/kubeflow/katib/utils/utils.py

```diff
@@ -40,6 +43,36 @@ def set_katib_namespace(katib):
     return namespace
 
 
+def has_condition(conditions, condition_type):
+    """Verify if the condition list has the required condition.
+    Condition should be valid object with `type` and `status`.
+    """
+
+    for c in conditions:
+        if c.type == condition_type and c.status == constants.CONDITION_STATUS_TRUE:
+            return True
+    return False
+
+
+def print_experiment_status(experiment: models.V1beta1Experiment):
```

Comment:
Consider implementing `V1beta1Experiment.__str__()`

Reply:
We have [auto generated methods for the `V1beta1Experiment` class](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/models/v1beta1_experiment.py#L212), is there any way to override them for our Katib client ?

Reply:
We can import our own mapping but I am not sure if it’s worth it. https://openapi-generator.tech/docs/customization/#bringing-your-own-models

Reply:
@terrytangyuan I guess, in such case we should define `V1betaExperiment` model by ourself, and make sure that all attributes and methods are properly set. Not sure if it worths to maintain.
WDYT @johnugeorge @tenzen-y ?

Reply:
I think the solution has more cons than pros since we must maintain the `V1betaExperiment` model ourselves.

Reply:
Let's not do that at this point unless we have more things to customize.